<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtAnt</class>
 <widget class="QWidget" name="QtAnt">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>954</width>
    <height>597</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtAnt</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>40</y>
     <width>51</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>测试界面</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_to_reset">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>200</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>复位指令</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>50</y>
     <width>111</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>天线标定</string>
   </property>
   <widget class="QPushButton" name="btn_to_decide_ant_direction">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>天线方位标定</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_to_decide_ant_pitch">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>天线俯仰标定</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>50</y>
     <width>111</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>天线状态</string>
   </property>
   <widget class="QPushButton" name="btn_inquire_ant">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>天线工作状态</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_inquire_gps">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>GPS状态</string>
    </property>
   </widget>
  </widget>
  <widget class="QLineEdit" name="line_direction">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>80</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_pitch">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>120</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>80</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>方向目标值</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>120</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>俯仰目标值</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_to_set_angle">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>150</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>设定目标角度</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_speed">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>240</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>200</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>天线轴</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>240</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>转动速度</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_to_set_speed">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>280</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>设定速度</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_to_stop_ant">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>240</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>停止运行</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_ant_axis">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>200</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_connect_tcp">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>490</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>TCP连接</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEdit">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>50</y>
     <width>341</width>
     <height>251</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_to_init_timer">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>430</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>定时器</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_ip">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>430</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_port">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>460</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>430</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>IP</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>460</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>PORT</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_to_set_rev">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>360</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>设置接收</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_to_realtieme">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>270</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>实时成像</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_to_delaytime">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>320</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>延时成像</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>340</y>
     <width>271</width>
     <height>221</height>
    </rect>
   </property>
   <property name="title">
    <string>地检</string>
   </property>
   <widget class="QPushButton" name="btn_to_rev_data_tran">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>160</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>连接接收</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>40</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>40</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>数据类型</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>120</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>帧长</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>80</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>参数值</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="frame_len">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>120</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>160</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>参数设置</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>80</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>QtAnt</receiver>
   <slot>btn_to_gather()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>38</x>
     <y>57</y>
    </hint>
    <hint type="destinationlabel">
     <x>46</x>
     <y>111</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>btn_to_gather()</slot>
  <slot>InitTimer()</slot>
 </slots>
</ui>
